src/include/cart/swim.h

```diff
@@ -128,6 +128,16 @@ struct swim_ops {
 	 */
 	int (*set_member_state)(struct swim_context *ctx,
 				swim_id_t id, struct swim_member_state *state);
+
+	/**
+	 * Increase the incarnation number of a given member.
```

Comment:
I think "Increase" is not appropriate word for this description. Maybe "Set new incarnation number of a given member." is better?

Reply:
Sure, will change to that.